hack/vendor.sh

```diff
@@ -36,27 +36,23 @@ clone() {
 	echo -n 'rm VCS, '
 	( cd $target_dir && rm -rf .{git,hg} )
 
+	echo -n 'rm vendor, '
+	( cd $target_dir && rm -rf vendor Godeps/_workspace )
+
 	echo done
 }
 
-clone git github.com/kr/pty 05017fcccf
-
+# the following lines are in sorted order, FYI
+clone git github.com/docker/libtrust 230dfd18c232
+clone git github.com/go-check/check 64131543e7896d5bcc6bd5a76287eb75ea96c673
+clone git github.com/go-fsnotify/fsnotify v1.2.0
 clone git github.com/gorilla/context 14f550f51a
-
 clone git github.com/gorilla/mux e444e69cbd
-
+clone git github.com/kr/pty 05017fcccf
 clone git github.com/tchap/go-patricia v2.1.0
-
 clone hg code.google.com/p/go.net 84a4013f96e0
-
 clone hg code.google.com/p/gosqlite 74691fb6f837
 
-clone git github.com/docker/libtrust 230dfd18c232
-
-clone git github.com/go-fsnotify/fsnotify v1.2.0
-
-clone git github.com/go-check/check 64131543e7896d5bcc6bd5a76287eb75ea96c673
```

Comment:
This whole section is just getting rid of whitespace and sorting the list so it's deterministic.  Zero functional changes here.